server/src/test/proto/spine/test/procman/commands.proto

```diff
@@ -52,3 +53,8 @@ message PmDoNothing {
 message PmThrowEntityAlreadyArchived {
     ProjectId project_id = 1;
 }
+
+message PmChangeLifecycle {
+    ProjectId project_id = 1;
+    spine.server.entity.LifecycleFlags lifecycleFlags = 2;
```

Comment:
The declaration style is wrong. It should NOT be camel-cased in the Protobuf definitions.

Also, can we please avoid having lifecycle flags as a parameter at all? Please have a command to archive and another to delete.

Reply:
Done.